src/base/memory_monitor_sr.cpp

```diff
@@ -50,11 +50,26 @@ namespace cubperf
     std::atomic<uint32_t> expand_resize_count;
   } MMON_MEM_STAT;
 
+  /* TODO: this dummy modules will be changed when heap module is registered */
+  MMON_STAT_INFO dummy_stat_info[] =
+  {
+    {MMON_STAT_DUMMY_1, NULL, NULL},
+    {MMON_STAT_DUMMY_2, "dummy_comp", "dummy_subcomp"},
+    {MMON_STAT_LAST, NULL, NULL} /* for recognizing end */
+  };
+
+  MMON_STAT_INFO long_dummy_stat_info[] =
+  {
+    {MMON_STAT_LONG_DUMMY_1, NULL, NULL},
+    {MMON_STAT_LONG_DUMMY_2, "LONGDUMMY_ATTR_CACHE", "LINKEDLIST"},
+    {MMON_STAT_LAST, NULL, NULL}
+  };
+
   class mmon_subcomponent
   {
     public:
       mmon_subcomponent (const char *subcomp_name)
-	: m_subcomp_name (subcomp_name) {}
+	: m_subcomp_name {subcomp_name}, m_cur_stat {0} {}
```

Comment:
I don't think we have a strict rule, but it's what I've been following. One initializer in a line.
```suggestion
	: m_subcomp_name {subcomp_name}
	, m_cur_stat {0} 
	{}
```

Reply:
Done.